### PR TITLE
Update registries-and-packs.md

### PR DESCRIPTION
**Pull request by antongisli**
WIP - as part of an attempt to consolidate terminology around packs, layers, and cluster profiles, this is a quick test PR based on https://docs.google.com/presentation/d/1w5PSUgyNH59Tz0AIPrBZ7UwxO9j5M7dZKdJhEIBHdr0/edit#slide=id.p31

Specifically, layers exist in cluster profiles, and are formed using packs. Also, the terms "core" and "infrastructure" packs are used interchangeably, and we should consolidate onto only one term. At present, my view is that will be "infrastructure". 

## Describe the Change

This PR .....

## Review Changes

💻 [Add Preview URL]()

🎫 [Jira Ticket]()

### CODE REVIEW

File: docs/docs-content/registries-and-packs/registries-and-packs.md [hunk 1]
```diff
@@ -13,19 +13,19 @@ sidebar_custom_props:
 # Packs
 
 
-A **Cluster Profile** is made up of preconfigured layers, each of which is called a pack.  In other words, **Packs** are the building block of a cluster profile to create layers such as operating systems, Kubernetes, network, storage, and add-ons. Packs can be broadly categorized into two types:
+A **Cluster Profile** is made up of layers, each of which is created using a pack.  In other words, **Packs** are the building block of a cluster profile to create layers such as operating systems, Kubernetes, network, storage, and add-ons. Packs can be broadly categorized into two types:
```
Comment (ritawatson) [inline docs/docs-content/registries-and-packs/registries-and-packs.md:16]: I suggest omitting the second sentence that starts "In other words...". The first sentence says it clearly, and the definition of Infrastructure packs is below.

File: docs/docs-content/registries-and-packs/registries-and-packs.md [hunk 2]
```diff
@@ -13,19 +13,19 @@ sidebar_custom_props:
...
 
 
-- **Add-On** packs - These packs model the infrastructure integrations and applications that exist on top of the core packs. Examples of applications are system, authentication, security, monitoring, logging, ingress, load balancer, service mesh, or helm charts. 
+- **Add-on** packs - These packs are used to form the integrations and application layers that exist on top of the infrastructure layers. Examples of applications are system, authentication, security, monitoring, logging, ingress, load balancer, service mesh, or helm charts. 
 
-Both the core and add-on packs described above are configurable, and you can define new add-on custom packs from scratch as well. The use case for defining new add-on packs is to have desired consistent governance across your profile deployments. 
+Both the infrastructure and add-on packs described above are configurable, and you can define new add-on custom packs from scratch as well. The use case for defining new add-on packs is to have desired consistent governance across your profile deployments.
```
Comment (ritawatson) [inline docs/docs-content/registries-and-packs/registries-and-packs.md:23]:
```suggestion
Both the infrastructure and add-on packs described above are configurable, and you can define new add-on custom packs from scratch as well. The use case for defining new add-on packs is to drive consistency across your profile deployments. 
```

File: docs/docs-content/registries-and-packs/registries-and-packs.md [hunk 3]
```diff
@@ -13,19 +13,19 @@ sidebar_custom_props:
...
 
-Both the core and add-on packs described above are configurable, and you can define new add-on custom packs from scratch as well. The use case for defining new add-on packs is to have desired consistent governance across your profile deployments. 
+Both the infrastructure and add-on packs described above are configurable, and you can define new add-on custom packs from scratch as well. The use case for defining new add-on packs is to have desired consistent governance across your profile deployments. 
   
 
 ## Pack Structure
 
-Palette provides a rich collection of out-of-the-box packs for various integrations and also offers extensibility through custom-built packs. To configure an existing pack (core or add-on) or to define a new add-on custom pack from scratch, it is essential to understand the pack structure. 
+Palette provides a rich collection of out-of-the-box packs for various integrations and also offers extensibility through custom-built packs. To configure an existing pack (infrastructure or add-on) or to define a new add-on custom pack from scratch, it is essential to understand the pack structure.
```
Comment (ritawatson) [inline docs/docs-content/registries-and-packs/registries-and-packs.md:28]:
```suggestion
Palette provides a rich collection of out-of-the-box packs for various integrations and also offers extensibility through custom-built packs. To configure an existing infrastructure or add-on pack or to define a new add-on custom pack, it is essential to understand the pack structure. 
```

Comment (ritawatson) [reply docs/docs-content/registries-and-packs/registries-and-packs.md:28]: Avoids parenthetical content.

**Review by ritawatson — APPROVED** [review]
Some minor suggestions. Otherwise LGTM.

**Comment by antongisli** [comment]
@ritawatson have finished accepting the changes

**Pull request merged**